test_conformance/images/kernel_read_write/test_read_1D.cpp

```diff
@@ -66,7 +66,7 @@ const char *float1DKernelSource =
 
 static const char *samplerKernelArg = " sampler_t imageSampler,";
 
-#define ABS_ERROR( result, expected ) ( fabsf( (float)expected - (float)result ) )
+#define ABS_ERROR(result, expected) (fabs((float)expected - (float)result))
```

Comment:
Hmm, we have several definitions of it. Is it worth pushing it to the common image code or the harness?

Reply:
Done.